spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -73,17 +59,13 @@
  * @author Mike Eltsufin
  */
 @Configuration
-@EnableConfigurationProperties({ SamplerProperties.class, GcpTraceProperties.class, SleuthProperties.class })
+@EnableConfigurationProperties(
+		{ SamplerProperties.class, GcpTraceProperties.class, SleuthProperties.class })
 @ConditionalOnProperty(value = "spring.cloud.gcp.trace.enabled", matchIfMissing = true)
 @ConditionalOnClass(TraceConsumer.class)
-@Import({ StackdriverTraceAutoConfiguration.TraceConsumerConfiguration.class,
-		StackdriverTraceAutoConfiguration.StackdriverTraceHttpAutoconfiguration.class })
-@AutoConfigureBefore({ TraceAutoConfiguration.class })
+@AutoConfigureBefore({ ZipkinAutoConfiguration.class })
```

Comment:
FYI `ZipkinAutoConfiguration` is already `@AutoConfigureBefore(TraceAutoConfiguration.class)`, which is why I think it's safe to remove it from here.